pkg/config/sink.go

```diff
@@ -464,12 +464,11 @@ type PulsarConfig struct {
 	// and 'type' always use 'client_credentials'
 	OAuth2 *OAuth2 `toml:"oauth2" json:"oauth2,omitempty"`
 
-	// Configure the service brokerUrl for the Pulsar service.
-	// This parameter from the sink-uri
-	brokerURL string `toml:"-" json:"-"`
-
-	// parse the sinkURI
-	u *url.URL `toml:"-" json:"-"`
+	// BrokerURL is used to configure service brokerUrl for the Pulsar service.
+	// This parameter is a part of the `sink-uri`. Internal use only.
+	BrokerURL string `toml:"-" json:"-"`
+	// U is the parsed sinkURI. Internal use only.
+	U *url.URL `toml:"-" json:"-"`
```

Comment:
```suggestion
	uri *url.URL `toml:"-" json:"-"`
```
The use of URI and URL needs to be uniform.